src/test/CMakeLists.txt

```diff
@@ -148,6 +148,7 @@ target_link_libraries(test_bitcoin
   bitcoin_cli
   bitcoin_node
   minisketch
+  secp256k1
```

Comment:
Would it make sense to update https://github.com/bitcoin/bitcoin/blob/master/src/secp256k1/CMakeLists.txt#L13-L22 now (i.e. we already mandate `CMAKE_VERSION 3.22`) - or is it important to keep this exactly as https://github.com/bitcoin-core/secp256k1/blob/master/CMakeLists.txt#L13?

Alternatively, should we synchronise the version in the two repos?

cc: @josibake

Reply:
> Alternatively, should we synchronise the versions in the two repos?

This happens on regular basis: https://github.com/hebasto/bitcoin/pulls?q=is%3Apr+Regular+rebasing

Reply:
> Would it make sense to update https://github.com/bitcoin/bitcoin/blob/master/src/secp256k1/CMakeLists.txt#L13-L22 now (i.e. we already mandate `CMAKE_VERSION 3.22`) - or is it important to keep this exactly as https://github.com/bitcoin-core/secp256k1/blob/master/CMakeLists.txt#L13?

We do not modify subtrees, such as `src/secp256k1`, `src/leveldb` etc.

Reply:
> Alternatively, should we synchronise the versions in the two repos?

Bitcoin Core is not the only consumer of the `libsecp256k1` library :)